From fcb24deb8b3abb8a77a12baa2cdd5ba5aa976f01 Mon Sep 17 00:00:00 2001
From: Michael Opdenacker <michael.opdenacker@bootlin.com>
Date: Wed, 8 Dec 2021 17:27:56 +0100
Subject: [PATCH] documentation: conf.py: fix version of bitbake objects.inv

Using the Bitbake 1.52 references instead of the master ones,
which may break if some variables or sections are removed.

Signed-off-by: Michael Opdenacker <michael.opdenacker@bootlin.com>
Reported-by: Quentin Schulz <foss@0leil.net>
---
 documentation/conf.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/documentation/conf.py b/documentation/conf.py
index 104b49ea0..eff758a7b 100644
--- a/documentation/conf.py
+++ b/documentation/conf.py
@@ -17,6 +17,7 @@ import sys
 import datetime
 
 current_version = "3.4"
+bitbake_version = "1.52"
 
 # String used in sidebar
 version = 'Version: ' + current_version
@@ -90,7 +91,7 @@ extlinks = {
 
 # Intersphinx config to use cross reference with Bitbake user manual
 intersphinx_mapping = {
-    'bitbake': ('https://docs.yoctoproject.org/bitbake/', None)
+    'bitbake': ('https://docs.yoctoproject.org/bitbake/' + bitbake_version, None)
 }
 
 # Suppress "WARNING: unknown mimetype for ..."
-- 
2.35.1

